src/styles/components/ec-datepicker/ec-datepicker.css

```diff
@@ -722,18 +722,19 @@ span.flatpickr-weekday {
 
 .flatpickr-day.flatpickr-disabled,
 .flatpickr-day.flatpickr-disabled:hover,
-.flatpickr-day.prevMonthDay,
-.flatpickr-day.nextMonthDay,
-.flatpickr-day.notAllowed,
-.flatpickr-day.notAllowed.prevMonthDay,
-.flatpickr-day.notAllowed.nextMonthDay {
+.flatpickr-day.notAllowed {
   @apply tw-text-gray-6; /* color: rgba(57,57,57,0.3); EC: replaced */
 
   background: transparent;
   border-color: transparent;
   cursor: default;
 }
 
+.flatpickr-day.prevMonthDay, 
+.flatpickr-day.nextMonthDay {
```

Comment:
what if next of prev month days are not allowed?

so the date has `class="flatpickr-day notAllowed prevMonthDay"` ? Will it be disabled or just `tw-text-gray-5`?

![image](https://user-images.githubusercontent.com/5939240/223969842-b5a79243-d063-45dc-8eb1-691d2dadf1f1.png)